packages/dd-trace/src/guardrails/telemetry.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+/* eslint-disable no-var */
+/* eslint-disable object-shorthand */
+
+var fs = require('fs')
+var spawn = require('child_process').spawn
+var tracerVersion = require('../../../../package.json').version
+var log = require('./log')
+
+module.exports = sendTelemetry
+
+if (!process.env.DD_INJECTION_ENABLED) {
+  module.exports = function () {}
+}
+
+if (!process.env.DD_TELEMETRY_FORWARDER_PATH) {
+  module.exports = function () {}
+}
+
+if (!fs.existsSync(process.env.DD_TELEMETRY_FORWARDER_PATH)) {
+  module.exports = function () {}
+}
+
+var metadata = {
+  language_name: 'nodejs',
+  language_version: process.versions.node,
+  runtime_name: 'nodejs',
+  runtime_version: process.versions.node,
+  tracer_version: tracerVersion,
+  pid: process.pid
+}
+
+var seen = []
+function hasSeen (point) {
+  if (point.name === 'abort') {
+    // This one can only be sent once, regardless of tags
+    return seen.includes('abort')
+  }
+  if (point.name === 'abort.integration') {
+    // For now, this is the only other one we want to dedupe
+    var compiledPoint = point.name + point.tags.join('')
+    return seen.includes(compiledPoint)
+  }
+  return false
+}
```

Comment:
@rochdev I just stumbled upon this while looking at some code.
It seems like `seen` is never changed throughout the code, so it's always going to return false. I wonder if that's critical or not. We should probably either remove this or fix it (after checking for includes, we add the entry to the seen array) and add tests?